src/animations/backwardsCompatibleAnimWrapper.js

```diff
@@ -0,0 +1,69 @@
+import {
+  always,
+  block,
+  call,
+  clockRunning,
+  cond,
+  set,
+  startClock,
+  stopClock,
+} from '../base';
+import { default as Clock } from '../core/AnimatedClock';
+import { default as Value } from '../core/AnimatedValue';
+
+function backwardsCompatibleInvoke(node, AnimationClass, value, config) {
+  let returnMethod;
+  const newValue = new Value(0);
+  const newClock = new Clock();
+  const currentState = AnimationClass.getDefaultState();
+  currentState.position = newValue;
+  let alwaysNode;
+  let isStarted = false;
+  return {
+    start: currentReturnMethod => {
+      if (isStarted) {
+        returnMethod && returnMethod({ finished: false });
+        return;
+      }
+      isStarted = true;
+      alwaysNode = always(
+        set(
+          value,
+          block([
+            cond(currentState.finished, set(currentState.finished, 0)),
+            cond(clockRunning(newClock), 0, [
+              set(newValue, value),
+              startClock(newClock),
+            ]),
+            node(newClock, currentState, config),
+            cond(currentState.finished, [
+              call([], () => {
+                isStarted = false;
+                alwaysNode.__removeChild(value);
+                returnMethod && returnMethod({ finished: true });
+              }),
+              stopClock(newClock),
+            ]),
+            currentState.position,
+          ])
+        )
+      );
+      returnMethod = currentReturnMethod;
+      alwaysNode.__addChild(value);
+    },
+    stop: () => {
+      isStarted = false;
+      alwaysNode.__removeChild(value);
+      returnMethod && returnMethod({ finished: false });
```

Comment:
I'm worried that under certain circumstances this could be called twice (here and in the `call` body defined above). I'd prefer if for `stop` method we just do `set(finished, 1)` then we could only have a single place that would handle "detaching"

Reply:
done